packages/cli/src/commands/start.ts

```diff
@@ -315,6 +316,8 @@ export class Start extends BaseCommand<z.infer<typeof flagsSchema>> {
 			);
 		}
 
+		await Container.get(LegacySqliteExecutionRecoveryService).cleanupWorkflowExecutions();
```

Comment:
- Should this be wrapped in a `isLegacySqlite` check so we skip this instantiation in all other cases?
- Should all mains clean up or only the leader?
- Should we lazy load this service so all other cases don't pay for this?